cmd/ako-main/main.go

```diff
@@ -117,7 +117,10 @@ func InitializeAKC() {
 		utils.AviLog.Warnf("Error in creating openshift clientset")
 	}
 
-	registeredInformers := lib.InformersToRegister(oshiftClient, kubeClient)
+	registeredInformers, err := lib.InformersToRegister(oshiftClient, kubeClient)
+	if err != nil {
+		utils.AviLog.Fatalf("Failed to initialize informers: %v, exiting AKO", err)
```

Comment:
Can we say - Shutting down AKO, going to reboot.

Reply:
Added